packages/@okta/vuepress-site/conductor.yml

```diff
@@ -2793,3 +2793,51 @@ redirects:
     to: /docs/guides/configure-signon-policy/main/#prompt-for-an-mfa-factor-when-a-user-is-outside-the-us
   - from: /docs/guides/configure-signon-policy/next-steps/index.html
     to: /docs/guides/configure-signon-policy/main/#next-steps
+  - from: /guides/sign-into-web-app/-/before-you-begin/index.html
```

Comment:
I believe you purposely left out the language-dependent stack selector from the "before-you-begin" section (and added that content generically under the Nutrition facts Sample heading), so, we don't need to have the hyphen language selector in the re-direct, nor can we reference "before-you-begin" in the To: section because that heading no longer exists. This re-direct can simply go to the top of the single page. I.e., the two re-directs should look like this:

`from: /docs/guides/sign-into-web-app/-/before-you-begin/index.html`
`to: /docs/guides/sign-into-web-app/`
`from: /docs/guides/sign-into-web-app/-/before-you-begin`
`to: /docs/guides/sign-into-web-app/`